charts/model-engine/values.yaml

```diff
@@ -5,7 +5,7 @@ redis:
   auth:
 db:
   runDbInitScript: false
-balloonNodeSelector:
-  node-lifecycle: normal
-nodeSelector:
-  node-lifecycle: normal
+# balloonNodeSelector:
```

Comment:
note- need to build out a good default values.yaml with comments specifying the purpose of each configuration option.